extensions/sql-migration/src/wizard/databaseSelectorPage.ts

```diff
@@ -212,9 +222,75 @@ export class DatabaseSelectorPage extends MigrationWizardPage {
 		// load unfiltered table list and pre-select list of databases saved in state
 		await this._filterTableList('', this.migrationStateModel._databasesForAssessment);
 
+		const xEventsDescription = this._view.modelBuilder.text()
+			.withProps({
+				value: constants.XEVENTS_ASSESSMENT_DESCRIPTION,
+				width: 650,
+				CSSStyles: {
+					...styles.BODY_CSS, 'width': '650px'
+				},
+				links: [{ text: constants.XEVENTS_ASSESSMENT_HELPLINK, url: 'aka.ms/sql-migration-xe-assess' }]
+			}).component();
+
+		const xEventsInstructions = this._view.modelBuilder.text()
+			.withProps({
+				value: constants.XEVENTS_ASSESSMENT_OPEN_FOLDER,
+				width: 650,
+				CSSStyles: { ...styles.LABEL_CSS, 'width': '650px' },
+			}).component();
+
+		this._xEventsFolderPickerInput = this._view.modelBuilder.inputBox()
+			.withProps({
+				placeHolder: constants.FOLDER_NAME,
+				readOnly: true,
+				width: 460,
+				ariaLabel: constants.XEVENTS_ASSESSMENT_OPEN_FOLDER
+			}).component();
+		this._disposables.push(
+			this._xEventsFolderPickerInput.onTextChanged(async (value) => {
+				if (value) {
+					this._xEventsFilesFolderPath = value.trim();
+				}
+			}));
+
+		const xEventsFolderPickerButton = this._view.modelBuilder.button()
+			.withProps({
+				label: constants.OPEN,
+				width: 80,
+			}).component();
+		this._disposables.push(
+			xEventsFolderPickerButton.onDidClick(
+				async () => this._xEventsFolderPickerInput.value = await promptUserForFolder()));
+
+		const xEventsFolderPickerClearButton = this._view.modelBuilder.button()
+			.withProps({
+				label: constants.CLEAR,
+				width: 80,
+			}).component();
+		this._disposables.push(
+			xEventsFolderPickerClearButton.onDidClick(
+				async () => {
+					this._xEventsFolderPickerInput.value = '';
+					this._xEventsFilesFolderPath = '';
+				}));
+
+		const xEventsFolderPickerContainer = this._view.modelBuilder.flexContainer()
+			.withProps(
+				{ CSSStyles: { 'flex-direction': 'row', 'align-items': 'left' } })
+			.withItems([
+				this._xEventsFolderPickerInput,
+				xEventsFolderPickerButton,
+				xEventsFolderPickerClearButton])
+			.component();
+
+		this._xEventsGroup = this._view.modelBuilder.groupContainer().withLayout({
+			header: constants.XEVENTS_ASSESSMENT_TITLE,
+			collapsible: true
+		}).withProps({ collapsed: false, width: 650, }).withItems([xEventsDescription, xEventsInstructions, xEventsFolderPickerContainer]).component();
```

Comment:
when initializing, please only set the collapsed property in the withLayout method.

Reply:
thanks @alanrenmsft, this worked